### ISSUE TITLE
Icon disappears from toolbar after update

### ISSUE THREAD

**Issue by pnsantos** [gorhill/uBlock#575]
Hi I've noticed this the last couple of updates that once uBlock Origin has been updated (without user intervention, i.e. just letting FF check for updates, download and update it automatically), until FF is restarted the icon dissappears from the toolbar (though uBlock Origin is still blocking stuff). Once I restart FF the icon comes back. Saw this in multiple computers too (one in Windows 7 x86 and in a OSX 10.10) all of them with the same plugin icons on the toolbar.

Currently using FF 39.0 on Win 8.1 x64 my current toolbar icons are (in this order):

- Home
- Lastpass (showing matching sites)
- uBlock Origin

Has anyone else seen this behavior? Is it normal or just a FF quirk/bug?

**Comment by Anahkiasen** [comment]
Not sure if it's related but on Chrome after each update uBlock disables itself and I have to reenable it.

**Comment by gorhill** [comment]
@Anahkiasen 

For Chrome, there is this open issue in Chromium issue tracker, which might be related to your specific issue: <https://code.google.com/p/chromium/issues/detail?id=515275>.

**Comment by StinVec** [comment]
I just switched to uBlock from AdBlock Edge today and was using FireFox 39.0.3 (I believe that was the last version number). There is an icon shell for the extension on the toolbar, but no graphic icon on the shell other than the little red square showing how many requests were blocked for the current page. Shortly afterward I updated to FireFox 40.0 and there is still no icon in the extension's icon shell.

I look forward to seeing what this extension's icon looks like in a future update that fixes this issue.
